data/json/items/melee/swords_and_blades.json

```diff
@@ -1730,5 +1730,75 @@
     "qualities": [ [ "AXE", 3 ] ],
     "use_action": [ "E_COMBATSAW_ON" ],
     "flags": [ "MESSY", "FRAGILE_MELEE", "TRADER_AVOID", "POWERED", "ALWAYS_TWOHAND" ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "punch_dagger",
+    "name": { "str": "punch dagger" },
+    "description": "A short and sharp double-edged dagger made to be gripped in the palm, with the blade protruding between the fingers.",
+    "weight": "168 g",
+    "to_hit": -1,
+    "price_postapoc": 50,
+    "color": "dark_gray",
+    "symbol": "{",
+    "material": [ "steel" ],
+    "volume": "250 ml",
+    "bashing": 2,
+    "cutting": 14,
+    "flags": [ "STAB", "DURABLE_MELEE", "SHEATH_KNIFE" ],
+    "price": 10000,
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 14 ] ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "knuckle_katar",
+    "symbol": "!",
+    "color": "dark_gray",
+    "name": { "str": "razorbar katar" },
+    "description": "Five pieces of rebar sharpened to cruel points and strapped to a brace fastened at wrist and forearm on both arms.  The next bub better watch out.",
+    "material": [ "steel" ],
+    "volume": "500 ml",
+    "weight": "1150 g",
+    "bashing": 3,
+    "cutting": 14,
+    "price_postapoc": 500,
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 8 ] ],
+    "flags": [ "STAB" ],
+    "techniques": [ "WBLOCK_1" ]
+  },
+  {
+    "type": "ARMOR",
+    "id": "cestus",
+    "symbol": "3",
+    "color": "light_gray",
+    "name": { "str": "cestus", "str_pl": "cestuses" },
+    "description": "A leather hand and arm wrap incorporating metal plates over the knuckles to improve punching power and defence.",
+    "material": [ "leather", "steel" ],
+    "volume": "100 ml",
+    "weight": "300 g",
+    "bashing": 2,
+    "price_postapoc": 250,
+    "flags": [ "UNARMED_WEAPON", "DURABLE_MELEE", "NONCONDUCTIVE" ],
```

Comment:
Cestus definitely isn't bladed weapon and still counts as unarmed.
Shouldn't be in this file